conformance_filters/core.csv

```diff
@@ -41,5 +41,3 @@ vector_load_store
 vector_operators
 vector_swizzle_assignment
 vector_swizzles
-vector_swizzles_opencl
-usm
```

Comment:
Why removing usm?

Reply:
I've returned "usm" in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/130/commits/16974d619d4543c9b1c0e2793bce2402e78a2d7f